### DISCUSSION TITLE
Missing linked data

### DISCUSSION THREAD

**Discussion by wstaelens** [schemaorg/schemaorg#3173] [General]
check it out! 

https://github.com/wstaelens/missing-linked-data

**Comment by danbri** [comment]
> Meanwhile search engines can use this as linked data to make the content
surface even better

Why would search engines trust these kinds of files as being something to
rely on for the main site? Not everyone will be as nice as you…

**Reply by wstaelens** [reply]
@danbri just trying to make data more accessible, as some archives will never see such data updates. It took me a few years just to move a person in 1 archive to add 10 metadata tags so that some important content would surface more 🙄 

I'm confident that some AI will check this 😄 

_(For example how does google know the json-ld is more trustworthy on a certain website compared to my files?)_
